compiler/control/OMROptions.cpp

```diff
@@ -2407,7 +2410,7 @@ OMR::Options::processOptionsJIT(char *jitOptions, void *feBase, TR_FrontEnd *fe)
       }
 
    if (_jitCmdLineOptions)
-      memset(_jitCmdLineOptions, 0, sizeof(TR::Options));
+      _jitCmdLineOptions = new (PERSISTENT_NEW) TR::Options();
```

Comment:
Isn't this going to result in us allocating the options object more than once?

Reply:
Indeed, this line need to be removed. _jitCmdLineOptions has been properly initialized by the constructor, on line 2408.

Reply:
Right, now that _jitCmdLineOptions is initialized properly, all that needs to be done is to set the _options array to all 0's. Thanks for catching that!